src/Modal/ModalDialog.jsx

```diff
@@ -25,11 +25,12 @@ function ModalDialog({
   className,
   isFullscreenOnMobile,
   isBlocking,
+  elevation,
 }) {
   const isMobile = useMediaQuery({ query: '(max-width: 767.98px)' });
   const showFullScreen = (isFullscreenOnMobile && isMobile);
   return (
-    <ModalLayer isOpen={isOpen} onClose={onClose} isBlocking={isBlocking}>
+    <ModalLayer isOpen={isOpen} onClose={onClose} isBlocking={isBlocking} elevation={elevation}>
```

Comment:
Similar to another comment, I wonder if there's a prop name other than `elevation` here to reduce potential confusion with shadows, etc.

What about `level` or `zIndex`?